### PR TITLE
[Validate-Great-Expectations] Update function to adhere to marketplace naming conventions

### PR DESCRIPTION
**Pull request by nschenone**

### CODE REVIEW

File: validate_great_expectations/item.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ mlrunVersion: 1.1.0
 name: great-expectations
```
Comment (yonishelach) [inline validate_great_expectations/item.yaml:17]: Name of the function need to be the same as the filename
```suggestion
name: validate-great-expectations
```

**Review by yonishelach — CHANGES_REQUESTED** [review]
Nice job, few things:
1. Please change the name of the directory to the function name as well.
2. After all the changes use (in command line): 
`python functions.py item-to-function -i validate_great_expectations` 
It will generate new `function.yaml`. Please make sure that the `project` field in the new `function.yaml`  is `''`.

File: validate_great_expectations/item.yaml (hunk 1 above)
Comment (nschenone) [reply validate_great_expectations/item.yaml:17]: Name of the function has been changed to `validate_great_expectations`. All file names have been changed accordingly

Comment (nschenone) [reply validate_great_expectations/item.yaml:17]: This was to prevent clashes between the function and the underlying Python library

Comment (nschenone) [reply validate_great_expectations/item.yaml:17]: Ok I see where you are talking about. Will make the changes

**Comment by nschenone** [comment]
@yonishelach Changes made 🙂

**Review by yonishelach — CHANGES_REQUESTED** [review]
Looks good, one last minor thing, please delete the old directory `great_expectations`

**Review by yonishelach — APPROVED** [review]

**Pull request merged**